db/migrations/v2.2.1.sql

```diff
@@ -1,27 +1,5 @@
-ALTER TABLE `user` ADD PRIMARY KEY(id)
+ALTER TABLE `project__user` ADD PRIMARY KEY(project_id);
```

Comment:
this primary key makes it impossible to add > 1 user to a project since the key structure is a unique key `user_id`+`project_id`